app/javascript/dashboard/assets/scss/widgets/_modal.scss

```diff
@@ -86,6 +86,12 @@
       margin-left: var(--space-small);
     }
 
+    &:first-child {
```

Comment:
<img width="407" alt="Screenshot 2023-03-24 at 3 49 53 PM" src="https://user-images.githubusercontent.com/2246121/227656933-b29010ab-b517-46f9-aad5-2c2da7c353b9.png">

To fix this.